asyncpg/connection.py

```diff
@@ -1644,6 +1683,11 @@ async def _executemany(self, query, args, timeout):
         executor = lambda stmt, timeout: self._protocol.bind_execute_many(
             stmt, args, '', timeout)
         timeout = self._protocol._get_timeout(timeout)
+
+        if self._query_logger:
+            self._query_logger.debug('Executemany query: %s', query)
+            self._query_logger.log(5, 'Arguments: %r', args)
```

Comment:
Number 5 here and in other places means `TRACE` level? Should be documented somewhere, like "To see more information (like query arguments) you need to call `getLogger("asyncpg.query").setLevel(5)`"

Reply:
Yes, that's `TRACE`. It isn't defined anywhere, so I just used a constant.
What would you suggest instead?

Reply:
Creating constant would be great, but if this will be documented - then 5 is fine by me :smile:

Reply:
Creating constant inside the library is discouraged: https://docs.python.org/3.8/howto/logging.html#custom-levels